pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/ClusterIntegrationTestUtils.java

```diff
@@ -766,11 +772,25 @@ public static void testQuery(@Nonnull String pinotQuery, @Nonnull String queryFo
           }
           String actualValue = actualValueBuilder.toString();
 
-          // Check actual value in expected values set
-          if (!expectedValues.contains(actualValue)) {
-            String failureMessage = "Selection result returned in Pinot but not in H2: " + actualValue;
-            failure(pinotQuery, sqlQueries, failureMessage);
-            return;
+          final BrokerRequest brokerRequest = PinotQueryParserFactory.get("pql").compileToBrokerRequest(pinotQuery);
+          final List<String> selectionColumns = brokerRequest.getSelections().getSelectionColumns();
+          final Set<String> orderByColumns = getOrderByColumns(brokerRequest.getOrderBy());
```

Comment:
Please use `brokerRequest.getSelections().getSelectionSortSequence()` as the orderByColumns